### ISSUE TITLE
Standalone Compatibility & suggestion

### ISSUE THREAD

**Issue by ELtd** [jetstreamlabs/zora#73]
Hi,

I came on this project thanks to your advice about translations and inertia on laracasts (https://laracasts.com/discuss/channels/inertia/translations-with-laravel-and-inertia?page=1&replyId=885512).

I followed your instructions, looking also at your serenity project, a bit of trial and errors and it works :)

But i had to change some things. For example : you refer to window.locale but this is not a standard. This constant comes from flowbite's date picker.
https://github.com/jetstreamlabs/zora/blob/2409188e8622013d4d6ffd9f3e7ab7f96585f754/dist/client.js#L4
Instead I use "document.documentElement.lang" to get the <html lang> value.

Also, I didn't want to have lodash in dependency, so i changed a bit the forEach to the classic version "replace.forEach((value, key){...})"
https://github.com/jetstreamlabs/zora/blob/2409188e8622013d4d6ffd9f3e7ab7f96585f754/dist/client.js#L36

Thanks for your work, it clears the way to go further 👍

**Comment by digvijayad** [comment]
Indeed. I'm currently trying to integrate it with svelte. I'll take a look at the serenity project you mentioned.

**Comment by digvijayad** [comment]
@secondmanveran I agree with keeping the dependencies in sync with Laravel.  However, lodash isn't a dependency on laravel 10.x versions. I just did a fresh laravel install using `composer create-project laravel/laravel example-app` and the content for package.json are as follows:
```json
{
    "private": true,
    "type": "module",
    "scripts": {
        "dev": "vite",
        "build": "vite build"
    },
    "devDependencies": {
        "axios": "^1.1.2",
        "laravel-vite-plugin": "^0.7.5",
        "vite": "^4.0.0"
    }
}
```
